README.md

```diff
@@ -1 +1,60 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> This project is about building a leaderboard website that displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- Code Editor (e.g. VS Code, Atom)
+- Web browser 
+
+### Setup
+
+ **Step 1: Clone this repo locally using git**
+- Create a folder in your local machine
+- Open your console in your folder and run this command: `https://github.com/dqarias/Leaderboard.git`
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+ **Step 2: Run in your project directory**
+- Open your console in your folder and run the nex command `. code` to open your vscode editor with awesomeBooks project.
+- Then in index.html you can open with live server.
+
```

Comment:
- [ ] Kindly also add the info on how to run your build and the dev server here in your README file as the third step.
[Hint]: The commands you need to mention `npm install`, `npm run build`, and `npm start`